src/Integration/src/IntegrationBase/Channel/MessagePublishingErrorHandler.cs

```diff
@@ -54,7 +54,7 @@ public bool HandleError(Exception exception)
             }
             catch (Exception errorDeliveryError)
             {
-                _logger?.LogWarning("Error message was not delivered.", errorDeliveryError);
+                _logger?.LogWarning("Error message was not delivered. {exception}", errorDeliveryError);
```

Comment:
Should pass the exception as the first parameter, such as [here](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Management/src/Endpoint/HeapDump/HeapDumper.cs#L74).